frontend/src/components/KanbanCard/KanbanCard.tsx

```diff
@@ -0,0 +1,47 @@
+import { Body, Button, P, Tooltip } from "@maykin-ui/admin-ui";
+import { FC } from "react";
+
+import "./KanbanCard.css";
+
+interface IKanbanCardProps {
+  title: string;
+  days?: string;
+  assigneeNames: string[];
+  downloadUrl?: string;
+}
+
+export const KanbanCard: FC<IKanbanCardProps> = ({
+  title,
+  days,
+  assigneeNames,
+  downloadUrl, // TODO: Implement download button, not in API yet so unsure how to pass as props, will withold UI implementation until clarification
+}) => {
+  const renderAssignees = () => {
+    if (assigneeNames.length > 1) {
+      const [first, ...rest] = assigneeNames;
+      return (
+        <>
+          <P>{first}</P>
+          <Tooltip content={rest.join(", ")} placement={"top"}>
+            <Button variant="transparent" pad={"h"}>
+              <P bold>+{rest.length}</P>
+            </Button>
+          </Tooltip>
+        </>
+      );
+    }
+    return assigneeNames[0];
+  };
+
+  return (
+    <Body className="kanban-card">
```

Comment:
Yeah, so the reason I have opted for this way is because Admin UI is not specific to JUST the open-archiefbeheer project. If I had instead implemented the Kanban Items in the admin-ui itself we would have some very specific "open-archiefbeheer" kanban cards, whereas now the Kanban cards can be very dynamic depending on the project. 
Implementing this in admin-ui would've us required to have props like: 
'days', 'asigneesCount', 'asigneesArray', 'title' etc. and I felt like this isn't the right path to go.